app/src/main/java/com/xinto/opencord/ui/LoginActivity.kt

```diff
@@ -1,26 +1,37 @@
 package com.xinto.opencord.ui
 
+import android.content.Context
 import android.os.Bundle
 import androidx.activity.compose.setContent
 import androidx.appcompat.app.AppCompatActivity
+import androidx.compose.foundation.isSystemInDarkTheme
 import androidx.compose.material3.MaterialTheme
 import androidx.compose.runtime.SideEffect
 import com.google.accompanist.systemuicontroller.rememberSystemUiController
 import com.hcaptcha.sdk.HCaptcha
+import com.hcaptcha.sdk.HCaptchaConfig
+import com.hcaptcha.sdk.HCaptchaTheme
 import com.xinto.opencord.ui.screen.LoginRootScreen
 import com.xinto.opencord.ui.theme.OpenCordTheme
 
 class LoginActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        HCaptcha.getClient(this).setup()
         setContent {
             OpenCordTheme {
                 val systemUiController = rememberSystemUiController()
                 val useDarkIcons = false
+                val useDarkTheme = isSystemInDarkTheme()
                 val surface = MaterialTheme.colorScheme.surface
 
+                val hCaptchaConfig = HCaptchaConfig.builder()
```

Comment:
I doubt it's a good idea to do this inside the composable scope, move it outside, just like how it was in line 17 previously.

Reply:
could it be injected?

Reply:
Quite possibly yes, the builder could be provided via DI and then setup could be run before setContent()

also inject deez nuts lmao gotem

Reply:
INJECT DEEZ NUTS lmao